scipy/special/__init__.py

```diff
@@ -618,6 +618,7 @@
    round   -- Round to nearest integer
    xlogy   -- Compute ``x*log(y)`` so that the result is 0 if ``x = 0``.
    xlog1py -- Compute ``x*log1p(y)`` so that the result is 0 if ``x = 0``.
+   logsumexp  -- Compute the log of the sum of exponentials of input elements.
```

Comment:
Indeed, `python setup.py install`, and then it works. Thanks for the heads-up, I missed the introduction of this helper.